frontend/src/metabase/query_builder/components/filters/pickers/SpecificDatePicker.tsx

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+
+import { getDateStyleFromSettings } from "metabase/lib/time";
+import Calendar, { SelectAll } from "metabase/components/Calendar";
+import InputBlurChange from "metabase/components/InputBlurChange";
+import Icon from "metabase/components/Icon";
+import ExpandingContent from "metabase/components/ExpandingContent";
+import HoursMinutesInput from "./HoursMinutesInput";
+
+import moment from "moment";
+import Filter from "metabase-lib/lib/queries/structured/Filter";
+import { TimeContainer } from "./SpecificDatePicker.styled";
+
+const DATE_FORMAT = "YYYY-MM-DD";
+const DATE_TIME_FORMAT = "YYYY-MM-DDTHH:mm:ss";
+
+export const getTimeComponent = (value: string) => {
```

Comment:
This file contains multiple components — can be a candidate for splitting into multiple files